libraries/AP_RangeFinder/AP_RangeFinder_LightWareSerial.cpp

```diff
@@ -33,7 +34,10 @@ AP_RangeFinder_LightWareSerial::AP_RangeFinder_LightWareSerial(RangeFinder::Rang
 {
     uart = serial_manager.find_serial(AP_SerialManager::SerialProtocol_Rangefinder, serial_instance);
     if (uart != nullptr) {
-        uart->begin(serial_manager.find_baudrate(AP_SerialManager::SerialProtocol_Rangefinder, serial_instance));
+        baudrate = serial_manager.find_baudrate(AP_SerialManager::SerialProtocol_Rangefinder, serial_instance);
+        uart->begin(baudrate);
+        // send enough serial transitions to trigger LW20 into serial mode
+        uart->write("www\r\n");
```

Comment:
should last_init_ms be set here as well?

Reply:
doesn't actually matter

Reply:
It doesn't, but we might be repeating the same actions every time it boots (it just depends on the `update` being called in less than one second or not).

Reply:
fixed